modules/apps/foundation/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/application/_dialog.scss

```diff
@@ -122,9 +128,21 @@
 		}
 	}
 
+	.blog-article-button-row,
```

Comment:
Hey @pat270 @jbalsas

I don't think it's a good idea that the theme needs to be aware of blogs, journal, or kb.... wouldn't it be possible to have a classed defined at the theme level that all apps use instead?

Thanks!